.npmrc

```diff
@@ -1,4 +1,4 @@
 always-auth=true
 registry=https://registry.npmjs.org/
 @cookiemonsterteam:registry=https://npm.pkg.github.com
-//npm.pkg.github.com/:_authToken=${GITHUB_REGISTERY_PAT}
+//npm.pkg.github.com/:_authToken=${GITHUB_REGISTRY_PAT}
```

Comment:
You can't change this without also changing the name of the key in `.github/workflows/CI.yml`.